src/test/rgw/bucket_notification/README.rst

```diff
@@ -0,0 +1,43 @@
+============================
+ Bucket Notification tests
+============================
+
+The tests use the Nose test framework. To get started, ensure you have
+the ``virtualenv`` software installed; e.g. on Debian/Ubuntu::
+
+        sudo apt-get install python-virtualenv
+
+and then run::
+
+        ./bootstrap
+
+You will need to use the sample configuration file named ``bntests.conf.SAMPLE`` 
+that has been provided at ceph/src/test/rgw/bucket_notification. This file can be
+used to run the bucket notification tests on a Ceph cluster started with vstart.
+
+You also need to install Kafka which can be done by downloading and unzipping from the following::
+
+        https://www.apache.org/dyn/closer.cgi?path=/kafka/2.6.0/kafka-2.6.0-src.tgz
+
+Then inside the kafka config directory you need to create a file named ``kafka_client_jaas.conf``
+with the following content::
+
+        KafkaClient {
+          org.apache.kafka.common.security.plain.PlainLoginModule required
+          username="alice"
+          password="alice-secret";
+        };
+
+After installing Kafka next is you need to start the Zookeeper and Kafka services.
+Here's the commands which can be used to start these services. For starting 
+Zookeeper service run::
+
+        bin/zookeeper-server-start.sh config/zookeeper.properties
+
+and then run to start the Kafka service::
+
+        bin/kafka-server-start.sh config/server.properties
+
+After starting vstart, zookeeper and kafka services you're ready to run the tests (when you're in /path/to/ceph/src/test/rgw/bucket_notification)::
+
+        BNTESTS_CONF=bntests.conf.SAMPLE ./virtualenv/bin/nosetests -s test_bn.py
```

Comment:
can you please change the instructions so that we don't run the tests from inside the dev tree